.env.example

```diff
@@ -9,6 +9,9 @@ AWS_BUCKET_NAME=
 WEBHOOK_AUTH_TOKEN=
 STORAGE_ENGINE=file
 VOTE_REPORT_SUBDIR=votes
+AI_SUMMARY_SUBDIR=ai-summary
+AI_TTS_SUBDIR=ai-tts
```

Comment:
Could we also add default values for these, if someone forget. also not changing in `.env.test`

Reply:
Default value is `undefined`, as defined here

https://github.com/snapshot-labs/snapshot-sidekick/blob/61a39b511596a7396c8aee2386f09fc99d529b9b/src/helpers/utils.ts#L41